tests/system_tests_qdstat.py

```diff
@@ -760,7 +774,19 @@ def get_ssl_args(self):
             return args
 
         def ssl_test(self, url_name, arg_names):
-            """Run simple SSL connection test with supplied parameters.
+            """
```

Comment:
The summary sentence of the docstring should be right after `""""`, on the same line https://www.python.org/dev/peps/pep-0008/#documentation-strings and linked https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

There are some linters for this in flake8 but they are disabled for Dispatch.